scheduler/conf.h

```diff
@@ -178,6 +178,8 @@ VAR gid_t		LogFileGroup		VALUE(0);
 					/* Group ID for log files */
 VAR cupsd_loglevel_t	LogLevel		VALUE(CUPSD_LOG_WARN);
 					/* Error log level */
+VAR int			StripUserDomain		VALUE(0);
+					/* Stripping domain after the local username (at auth.c:1725) */			
```

Comment:
Don't specify a line number or a source file in the comment - that will become easily out of date. Better to say "Strip domain in local username?" and leave it at that. The man page will explain things in more depth plus we have comments where this is used.